lib/ASRouter.jsm

```diff
@@ -518,7 +518,8 @@ class _ASRouter {
         if (!msg.targeting) {
           continue;
         }
-        const isMatch = await ASRouterTargeting.isMatch(msg.targeting);
+        const context = this._getMessagesContext();
```

Comment:
On second thought this context should be independent of messages and should be safe to call once, updating patch with the change

Reply:
Done